index.html

```diff
@@ -19,9 +18,45 @@
     <link href="./styles/main.css" rel="stylesheet" />
     <link href="https://iacr.org/libs/fonts/raleway-regular.css" rel="stylesheet"/>
     <link href="https://iacr.org/libs/fonts/open-sans-condensed.css" rel="stylesheet"/>
+    <style>
+     .sessionlist {
+       margin-top: 3px;
+       margin-bottom: 3px;
+     }
+     .session, .tab-content > .active {
+       background-color: #dee2e6;
+     }
+     .nav-tabs .nav-item.show .nav-link, .nav-tabs .nav-link.active {
+       background-color: #dee2e6;
+     }
+     .sessionlist .nav-tabs .nav-link {
+       border-bottom: none;
+     }
+     .session:first-child:nth-last-child(1) {
+       /* -or- .session:only-child { */
+       width: 100%;
+     }
+     /* two sessions */
+     .session:first-child:nth-last-child(2),
+     .session:first-child:nth-last-child(2) ~ .session {
+       width: 50%;
+     }
+
+     /* three sessions */
+     .session:first-child:nth-last-child(3),
+     .session:first-child:nth-last-child(3) ~ .session {
+/*       width: 33.3333%;*/
+     }
+
+     /* four sessions */
+     .session:first-child:nth-last-child(4),
+     .session:first-child:nth-last-child(4) ~ .session {
+       width: 25%;
```

Comment:
not technically commented out but since we're removing it anyway, perhaps it doesn't matter

Reply:
We can remove the stuff for three or four sessions in the next revision.